### DISCUSSION TITLE
excuse me. where is the `.icon` filefolder

### DISCUSSION THREAD

**Discussion by qtw1998** [squidfunk/mkdocs-material#3066] [Show and tell]
![image](https://user-images.githubusercontent.com/34357479/135466446-b4336393-bb67-4686-8740-c24867a60f31.png)
I wanna change the `svg` file in the `main.html`
waiting for your help thx!!

**Comment by squidfunk** [comment]
https://github.com/squidfunk/mkdocs-material/tree/master/material/.icons
